TramsDataApi/RequestModels/AcademyConversionProject/UpdateAcademyConversionProjectRequest.cs

```diff
@@ -58,7 +58,10 @@ public class UpdateAcademyConversionProjectRequest
         // pupil schools forecast
         public string SchoolPupilForecastsAdditionalInformation { get; set; }
 
-        //key stage performance tables
+        // key stage performance tables
         public string KeyStagePerformanceTablesAdditionalInformation { get; set; }
+
+        // project notes
+        public ProjectNoteRequest ProjectNote { get; set; }
```

Comment:
I think project notes shouldn't be bundled in the with AcademyConversionProject request model as they are logically separate entities. Everything in the ACP is intended to go into the Htb template and has its own whereas the project notes only appear in the project notes tab. I think having a separate POST method on the controller (or separate controller) that takes a project note would make sense as they have their own table